src/Rest/Client.php

```diff
@@ -133,7 +133,7 @@ protected function send($path, array $query = [], $data = NULL, array $options =
     if (!empty($result->error)) {
       throw new HttpError($result);
     }
-    return drupal_json_decode($result->data);
+    return json_decode($result->data, TRUE, 512, JSON_THROW_ON_ERROR);
```

Comment:
I think we shouldn’t expect there to always be data in the response? Consider 201 or 202 responses, for example. Empty `$result->data` would now throw an error instead of returning NULL.

Reply:
True. I’ve added support for empty responses in a new commit.